packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -14,7 +14,7 @@ import resolvable, { Resolvable } from '@josephg/resolvable';
 import {
   InMemoryLRUCache,
```

Comment:
So you allude to this in the PR description but... isn't this a backwards-incompatible change, from an unbounded to a bounded default cache?

Could fix that by using the default (Map-backed) keyv. Or by putting an extraordinarily large max/maxSize that is still finite...

Reply:
Agree, I brought back keyv since it's actually unbounded and we weren't even leveraging size calculation in the default case. Let me know what you think.
[97dcb80](https://github.com/apollographql/apollo-server/pull/6522/commits/97dcb803a8d5dea21232043da72c2a0702ec8ccd)

My only hesitation is introducing the keyv dependency, but in this case we're just a consumer and not implementing our own so maybe less concerning.

Reply:
Hmm. You know, your test TTL cache made it kinda clear to me that it would be very easy to just implement our own ttl-obeying unbounded cache. Just store the value and the "deadline" (now + TTL) in a map, check time at read time. Maybe that beats the dependency on keyv.  Though at least it's a purely internal dependency on keyv rather than something part of the API...